src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-bottom: 5px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 22px;
+    &_stars {
+      @include stars;
+    }
+
+    &_count {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    &_title {
+      color: $second-text-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+    &_sum {
+      &::after {
+        content: '$#{$price-in-string}';
+        font-weight: 700;
+        color: $main-text-color;
+        line-height: 18px;
+      }
+    }
+  }
+
+  &__button {
+    display: block;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    background-color: $button-color;
+    text-transform: uppercase;
+    text-align: center;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 38px;
+    &:hover {
+      color: $button-color;
```

Comment:
Ensure that the variable `$button-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.